employees/models.py

```diff
@@ -1,10 +0,0 @@
-from django.db import models
-from positions.models import Position
-
-
-# Create your models here.
-class Employee(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
__str__ function is not declared for the admin